### PR TITLE
chore: add logs to help with debugging

### PR DESCRIPTION
**Pull request by lwhiteley**
If statement still not executing for some reason

can we debug and ensure its working before its released again?

github actions like these are annoying to test because they have to be on the default branch

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -117,6 +117,9 @@ ${formatImage({
     ? `https://github.com/${github.context.repo.owner}/${github.context.repo.repo}/runs/${checkRunId}`
     : `https://github.com/${github.context.repo.owner}/${github.context.repo.repo}/actions/runs/${github.context.runId}`;
 
+  core.info(`teardown enabled?: ${teardown}`);
+  core.info(`event action?: ${payload.action}`);
```
Comment (afc163) [inline src/main.ts:121]:
```suggestion
  core.debug(`event action?: ${payload.action}`);
```

**Comment by afc163** [comment]
One more `"` in image.

**Comment by lwhiteley** [comment]
updated

File: src/main.ts [hunk 2]
```diff
@@ -117,6 +118,9 @@ ${formatImage({
     ? `https://github.com/${github.context.repo.owner}/${github.context.repo.repo}/runs/${checkRunId}`
     : `https://github.com/${github.context.repo.owner}/${github.context.repo.repo}/actions/runs/${github.context.runId}`;
 
+  core.info(`teardown enabled?: ${teardown}`);
```
Comment (afc163) [inline src/main.ts:121]:
```suggestion
  core.debug(`teardown enabled?: ${teardown}`);
```

Comment (lwhiteley) [reply src/main.ts]: updated 😸

**Pull request merged**